hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -513,27 +409,6 @@ public boolean rename(final Path src, final Path dst) throws IOException {
         listener);
 
     // Non-HNS account need to check dst status on driver side.
-    if (shouldRedirect(FSOperationType.RENAME, tracingContext)) {
-      Path wasbSrc = src;
-      Path wasbDest = dst;
-      if (src.toString().contains(FileSystemUriSchemes.ABFS_SCHEME)
-          || src.toString().contains(FileSystemUriSchemes.ABFS_SECURE_SCHEME)) {
-        wasbSrc = new Path(abfsUrlToWasbUrl(src.toString(),
-            abfsStore.getAbfsConfiguration().isHttpsAlwaysUsed()));
-      }
-      if (dst.toString().contains(FileSystemUriSchemes.ABFS_SCHEME)
-          || dst.toString().contains(FileSystemUriSchemes.ABFS_SECURE_SCHEME)) {
-        wasbDest = new Path(abfsUrlToWasbUrl(dst.toString(),
-            abfsStore.getAbfsConfiguration().isHttpsAlwaysUsed()));
-      }
-      try {
-        return nativeFs.rename(wasbSrc, wasbDest);
-      } catch (IOException e) {
-        LOG.debug("Rename failed ", e);
-        throw e;
-      }
-    }
-    else {
       trailingPeriodCheck(dst);
 
```

Comment:
the code in else, might have to fixed for whitespace.

Reply:
else is not needed so removed that part